PayPal.podspec

```diff
@@ -39,6 +39,11 @@ Pod::Spec.new do |s|
     s.resource_bundle = { "PayPalWebPayments_PrivacyInfo" => "Sources/PayPalWebPayments/PrivacyInfo.xcprivacy" }
   end
 
+  s.subspec "VenmoPayments" do |s|
+    s.source_files  = "Sources/VenmoPayments/*.swift"
```

Comment:
Looking at the files it seems like the `VenmoPayments` directory is not currently in `Sources`

Reply:
addressed in b1a40ac05164a7165d03e91f862683de837141ca

Reply:
tyyy, addressed in b1a40ac05164a7165d03e91f862683de837141ca